carapace-server/src/main/java/org/carapaceproxy/server/HttpProxyServer.java

```diff
@@ -216,27 +229,32 @@ public void startAdminInterface() throws Exception {
         context.addServlet(jerseyServlet, "/api/*");
         context.addServlet(new ServletHolder(new MetricsServlet()), "/metrics");
         context.setAttribute("server", this);
-        contexts.addHandler(context);
-
-        File webUi = new File(basePath, "web/ui");
-        if (webUi.isDirectory()) {
-            WebAppContext webApp = new WebAppContext(webUi.getAbsolutePath(), "/ui");
-            contexts.addHandler(webApp);
-        } else {
-            LOG.severe("Cannot find " + webUi.getAbsolutePath() + " directory. Web UI will not be deployed");
-        }
+        
+        NCSARequestLog requestLog = new NCSARequestLog();
+        requestLog.setFilename(adminAccessLogPath);
```

Comment:
Where are we writing this logs in tests? 
Are we polluting the project directory?

Reply:
They are wrote in a temp dir (junit TemporaryFolder)